gce-disk-expand/packaging/gce-disk-expand.spec

```diff
@@ -30,23 +31,25 @@ gce-disk-expand: Automatically resize the root partition on first boot.
 This package is adopted from cloud-utils, cloud-initramfs-tools, and
```

Comment:
I think we can remove this description now- its no longer adopted from these packages.